src/hidimstat/distilled_conditional_randomization_test.py

```diff
@@ -109,44 +111,35 @@ def __init__(
         method: str = "predict",
         estimated_coef=None,
         sigma_X=None,
-        params_lasso_screening={
-            "alpha": None,
-            "n_alphas": 10,
-            "alphas": None,
-            "alpha_max_fraction": 0.5,
-            "cv": 5,
-            "tol": 1e-6,
-            "max_iter": 1000,
-            "fit_intercept": False,
-            "selection": "cyclic",
-        },
-        params_lasso_distillation_x=None,
+        lasso_screening=LassoCV(n_alphas=10, cv=5, tol=1e-6, fit_intercept=False),
+        model_distillation_x=LassoCV(n_jobs=1, n_alphas=10),
```

Comment:
```suggestion
        lasso_screening=LassoCV(n_alphas=10, cv=5, tol=1e-6, fit_intercept=False, random_state=0),
        model_distillation_x=LassoCV(n_jobs=1, n_alphas=10, random_state=0),
```
I prefer to fix the random generator for the default parameters.